source/howtos/dynamic_services.rst

```diff
@@ -24,7 +24,7 @@ dynamic service, add the following to the kbase.yml file:
     service-config:
         dynamic-service: true
 
-You can then register your module as usual and start and stop it in a production environment using the `catalog interface`_ as described below in the :ref:`Manage dynamic services in a KBase environment section <in-kbase>`.
+You can then register your module as usual and start and stop it in a production environment using the `catalog interface`_ as described below in the :ref: `Manage dynamic services in a KBase environment section <in-kbase>`.
```

Comment:
Adding a space between `:ref:` and `\`Manage dynamic services...` breaks the link and formatting for me.